golos.social/golos.social.cpp

```diff
@@ -48,6 +48,19 @@ void social::unpin(name pinner, name pinning) {
     });
 }
 
+void social::removepin(name pinner, name pinning) {
```

Comment:
We do not need an additional method. Cleaning the table can be carried out in the `unpin/unblock` methods.
The only thing is to make a separate function check that the record can be deleted.